_includes/tutorials/console-consumer-produer-basic/kafka/markup/dev/produce-topic.adoc

```diff
@@ -0,0 +1,16 @@
+To produce your first record into Kafka, open a terminal window an run the following command:
+
++++++
+<pre class="snippet"><code class="shell">{% include_raw tutorials/console-consumer-produer-basic/kafka/code/tutorial-steps/dev/console-producer.sh %}</code></pre>
++++++
+
+The producer will start and wait for you to enter input.  Each line represents one record and to send it you'll hit the enter key.  If you type multiple words then hit enter, the entire line is considered one record.
```

Comment:
Is it worth explaining keys & separators, for produce?

Reply:
Yeah, but I think it's covered in _includes/tutorials/console-consumer-produer-basic/kafka/markup/dev/produce-topic-keys-values.adoc, is the coverage there enough